week-3/Homework/mandatory/6-lane-names.js

```diff
@@ -4,43 +4,45 @@
   Write a function that will return all street names which contain 'Lane' in their name.
 */
 
-function getLanes() {
-
+function getLanes(arr) {
+  let result = arr.filter((getLanes) => getLanes.includes("Lane"));
+  return result;
 }
```

Comment:
Nice and neat solution! You could even make it neater by returning the array without assigning it to `result` variable 😉 I wouldn't recommend naming array elements processed by `filter` method the same as the main function name to avoid confusion